pkg/controllers/management/node/controller.go

```diff
@@ -403,6 +403,12 @@ func aliasToPath(driver string, config map[string]interface{}, ns string) error
 				if err != nil {
 					return err
 				}
+				if !devMode && settings.UnprivilegedJailUser.Get() == "true" {
+					err = jailer.SetJailOwnership(fullPath)
```

Comment:
Could we check if assigning ownership to files other than `userdata` cause issues?

Reply:
I'll need QA's help to test it on vSphere tomorrow.